recipe/build.sh

```diff
@@ -2,27 +2,25 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-if [[ "$OSTYPE" == "msys" ]]; then
-    mathjax="$(cygpath "${LIBRARY_LIB}")/mathjax"
-    bin_dir="$(cygpath "$SCRIPTS")"
-else
-    mathjax="$PREFIX/lib/mathjax"
-    bin_dir="$PREFIX/bin"
-fi
-
-
-
-
-
+mathjax="$PREFIX/lib/mathjax"
 mkdir -p "$mathjax"
-mv ./es5 "$mathjax/"
-
-
 
-mkdir -p "${bin_dir}"
-cp "${RECIPE_DIR}/mathjax-path" "${bin_dir}/"
-if [[ "$OSTYPE" == "msys" ]]; then
-    cp "${RECIPE_DIR}/mathjax-path.bat" "${bin_dir}/"
-else
-    chmod +x "${bin_dir}/mathjax-path"
-fi
+mv config "$mathjax/"
```

Comment:
Wouldn't it be simpler to just delete what needs to be deleted adn then copy/move everything else (i.e. `cp -r . "${mathjax}/"`)?

So basically
```
mkdir -p "$PREFIX/lib/mathjax"

rm -f *.sh LICENSE *.md .gitignore .travis.yml bower.json composer.json latest.js package.json

cp -r . "${mathjax}/"

mkdir -p "$PREFIX/bin"
cp "${RECIPE_DIR}/mathjax-path" "$PREFIX/bin/"
chmod +x "$PREFIX/bin/mathjax-path"
```

Reply:
At the current location there are also conda related scripts such as conda_build.sh for example. I don't want the script to remove/move itself. I also think it is nice to see what it is actually copying/moving.